data/mods/Generic_Guns/gg_spare_override.json

```diff
@@ -0,0 +1,23 @@
+[
```

Comment:
I wasn't sure how best to name it.
These were guns which presented errors since no 50/44 ammo types were available.
I think it may be because they use copy-from to guns which the mod overrides?
If you have a better suggestion, I would appreciate it!